README.md

```diff
@@ -73,6 +83,10 @@ import Client from 'shopify-buy/index.amd';
 ```javascript
 import Client from 'shopify-buy/index.umd';
 ```
+**UMD Unoptimized:**
```

Comment:
You should maybe include a definition of what unoptimized means. The term is very inside baseball as it is.

Reply:
At least in terms of the cases where this version is useful.